platform/macos/export/export_plugin.cpp

```diff
@@ -724,21 +724,14 @@ Error EditorExportPlatformMacOS::_code_sign(const Ref<EditorExportPreset> &p_pre
 			args.push_back(p_path);
 
 			String str;
-			int exitcode = 0;
 
 			Error err = OS::get_singleton()->execute("codesign", args, &str, nullptr, true);
```

Comment:
```suggestion
			Error err = OS::get_singleton()->execute("codesign", args, &str, &exitcode, true);
```
And keep the rest of the code as is.

Reply:
Thanks for your comments. Done.